lib/funneler/version.rb

```diff
@@ -0,0 +1,3 @@
+module Funneler
+  VERSION = "1.0"
```

Comment:
Yeah, but I think we need an extra version number `.0` to conform to semver and allow us to cut "patch releases" :smile: